extensions/cornerstone/src/services/CornerstoneCacheService/CornerstoneCacheService.ts

```diff
@@ -186,9 +201,22 @@ class CornerstoneCacheService {
           displaySet,
           dataSource
         );
+        const distributedCopy = (items, n) => {
```

Comment:
This is going to cause rendering issues when items.length ~= n because it is just going to throw out a certain set of instances, which causes the interval between images to be off by a factor of 2.  When n << items.length, it works ok,
because the interval becomes nearly consistent again, or if n = items.length / integer value, then one can sub-select evenly.  It is possible to write a inconsistent sample viewer, but currently we assume in the configuration that the sample volume is consistent, and don't account for this type of differences.

Reply:
You could write a decimator instead - that takes out every other image, or as a general implementation, leaves 1 image, takes k images out, then repeats.  Decimators leave consistent spacing.

Reply:
I added a decimator to CS3D in the HTJ2K progressive branch if you want one.